docs/developer/app-store/apps/emails-and-messages/overview.mdx

```diff
@@ -0,0 +1,122 @@
+
+1. Open the Saleor dashboard
+1. Navigate to the Configuration page
+1. Click on the Site settings card
+1. Update the `User registration` form
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account password reset
+
+Message sent after requesting a password change. The payload contains a token that has to be attached to the request to the Saleor API to change the password. To implement a storefront interface for password change, follow [the documentation](../../../users#resetting-the-password).
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account password reset for accounts created by the staff
+
+As in the previous event, but for accounts created by Staff users in the Saleor Dashboard.
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account email address change
+
+Message sent on the email address change request. Follow the [documentation](../../../users#changing-the-email-address) for more details.
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account email change confirmation
+
+Confirmation is sent to the new address after following directions from the previous message.
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account deletion request
+
+A message with a token that's used for account deletion. To implement a storefront interface for account deletion, follow [the documentation](../../../users#deleting-the-account).
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Invoice sent
+
+The event is triggered when the Staff users use the `Sent invoice` action. The message contains a link to the invoice.
+
+The event has been implemented using the `INVOICE_SENT` webhook.
+
+### Order has been confirmed
+
+Sends a message when the order is confirmed.
+
+The event has been implemented using the `ORDER_CONFIRMED` webhook.
+
+### Order has been fulfilled
+
+Sends a message when the order status is changed to `FULFILLED`.
+
+The event has been implemented using the `ORDER_FULFILLED` webhook.
+
+### Order has been fully paid
+
+Sends a message when the order is fully paid.
+
+The event has been implemented using the `ORDER_FULLY_PAID` webhook.
+
+### Order has been canceled
+
+Sends a message when the order is canceled.
+
+The event has been implemented using the `ORDER_CANCELLED` webhook.
+
+### Sent gift card
+
+Sends a message when the gift card is sent to the customer.
+
+The event has been implemented using the `GIFT_CARD_SENT` webhook.
+
+## Configuration
+
+The application can use multiple providers at the same time. This way, you'll be able to:
+
+- create test configurations before using them in production
+- use different providers for different events
+- use different providers for different channels
+
+### Multichannel
+
+Each provider can be configured to work with specific channels. The configurations by default are used for all channels. It can be changed in the configuration details to:
```

Comment:
**polish**: IMO, it's a more natural sentence structure.

```suggestion
Each provider can be configured to work with specific channels. The configurations are used for all the channels by default. It can be changed in the configuration details to:
```